language-agent/JVMMetric.proto

```diff
@@ -85,8 +86,22 @@ enum GCPhrase {
 }
 
 message Thread {
-  int64 liveCount = 1;
-  int64 daemonCount = 2;
-  int64 peakCount = 3;
+    int64 liveCount = 1;
+    int64 daemonCount = 2;
+    int64 peakCount = 3;
+    int64 deadlocked = 4;
+    int64 monitorDeadlocked = 5;
+    int64 newThreadCount = 6;
+    int64 runnableThreadCount = 7;
+    int64 blockedThreadCount = 8;
+    int64 waitThreadCount = 9;
+    int64 timeWaitThreadCount = 10;
+    int64 terminatedThreadCount = 11;
+}
+
+message Class {
+    int64 loadedClassCount = 1;
+    int64 unloadedClassCount = 2;
```

Comment:
```suggestion
    int64 totalUnloadedClassCount = 2;
```
![image](https://user-images.githubusercontent.com/28091237/124240907-d0624d00-db4d-11eb-8195-0364d8167c39.png)

Reply:
resolve, please review it.